commitlint.config.js

```diff
@@ -0,0 +1,68 @@
+import {
+	RuleConfigSeverity,
+} from '@commitlint/types';
+
+const Configuration = {
+    extends: ['@commitlint/config-conventional'],
+    rules: {
+        'type-enum': [
+			RuleConfigSeverity.Error,
+			'always',
+			[
+				'breaking-change',
```

Comment:
Consider pulling all valid types and scopes from https://raw.githubusercontent.com/datalens-tech/datalens/main/.github/workflows/scripts/changelog/changelog_config.json